config/_default/config.toml

```diff
@@ -181,3 +181,13 @@ navbar_logo = false
 	url = "https://review.gerrithub.io/plugins/gitiles/cue-lang/cue"
 	icon = "fab fa-git"
         desc = "The source-of-truth repo."
+[module]
```

Comment:
Why is this change needed? 

Is this part of a switch to docsy using Go modules as opposed to git sub modules?

If so, does the submodule itself need to be removed now?

Reply:
If moving to v0.1.0 works for you, that seems like a better change to make now. 

We can of course revisit upgrading further in a later PR.